src/gui/data_structures/frame_buffer_multisample.cpp

```diff
@@ -15,8 +20,8 @@ gui::data_structures::FrameBufferMultisample::FrameBufferMultisample(
 
     // frame buffer (the container for the other two)
     glGenFramebuffers(1, &frame_buffer);
-    gui::FrameBufferHandler::bind_fbo(frame_buffer);
-
+    gui::FrameBufferHandler* frame_buffer_handler;
+    frame_buffer_handler->getInstance().bind_fbo(frame_buffer);
```

Comment:
What the hell?

Alem its a static method.

This is so unbelievably cursed

```suggestion
    gui::FrameBufferHandler::getInstance().bind_fbo(frame_buffer);
```